build/build.ps1

```diff
@@ -250,12 +250,24 @@ function Build {
     }
     finally
     {
-      # Kill compiler server and MSBuild node processes from bootstrapped MSBuild (otherwise a second build will fail to copy files in use)
-      foreach ($process in Get-Process | Where-Object {'msbuild', 'dotnet', 'vbcscompiler' -contains $_.Name})
-      {
-        if ($process.Path.StartsWith( $RepoRoot, [StringComparison]::InvariantCultureIgnoreCase))
+      # Jenkins does not allow taskkill
```

Comment:
Builds [fail](https://ci2.dot.net/job/Microsoft_msbuild/job/exp_update-toolset/job/innerloop_Windows_NT_CoreCLR_prtest/27/console) with what I assume is taskkill not being supported: 
```
17:23:14 ERROR: The process with PID 4396 could not be terminated.
17:23:14 Reason: The operation attempted is not supported.
```

Same thing with !windows builds where `kill` is not supported.

Reply:
Huh. Maybe because sometimes multiple Jenkins agents run on a single physical machine, so they take precautions to prevent you from horking the other build?

Reply:
Found the !windows [equivalent](https://ci2.dot.net/job/Microsoft_msbuild/job/exp_update-toolset/job/innerloop_OSX10.13_CoreCLR_prtest/8/console):

```
16:15:13 /Users/dotnet-bot/j/w/Microsoft_msbuild/exp_update-toolset/innerloop_OSX10.13_CoreCLR_prtest/build/build.sh: line 315: kill: (172) - Operation not permitted
16:15:13 ./build/cibuild.sh: line 11: 11117 Terminated: 15          bash $ScriptRoot/build.sh -fromScript $0 -build -ci -prepareMachine $*
16:15:14 FATAL: command execution failed
16:15:14 java.nio.channels.ClosedChannelException
```

Reply:
Sounds plausible.